packages/@aws-cdk/cloud-assembly-schema/lib/assets/docker-image-asset.ts

```diff
@@ -80,6 +80,14 @@ export interface DockerImageSource {
    * @default - current machine platform
    */
   readonly platform?: string;
+
+  /**
+   * Outputs
+   *
+   * @default - No outputs
```

Comment:
Let's be a bit more descriptive here. It's also okay to link to the docker docs with `@see`

Reply:
@jesse-peters Once you've update this doc block and re-generated the schema, the PR should be good to go.

Reply:
@mrgrain I believe I pushed this last night?

Reply:
Yep I saw that. This is a different place that you've probably just missed. Unfortunately there's a lot of repetition with this. =/

Reply:
Oops, you're right! I just pushed an update.